hpx/hpx_init_params.hpp

```diff
@@ -0,0 +1,83 @@
+//  Copyright (c)      2020 CSCS - ETH Zurich
```

Comment:
Thanks for doing this! :D I think it should be just `ETH Zurich` and we should consider going back to change all our contributions to be `ETH Zurich` as well. Let's start with this one though...